docs/apache-airflow/index.rst

```diff
@@ -98,14 +98,14 @@ Rich scheduling and execution semantics enable you to easily define complex pipe
 intervals. Backfilling allows you to (re-)run pipelines on historical data after making changes to your logic.
 And the ability to rerun partial pipelines after resolving an error helps maximize efficiency.
 
-Airflow's user interface provides both in-depth views of pipelines and individual tasks, and an overview of
+Airflow's user interface provides both in-depth views of pipelines, and individual tasks and an overview of
 pipelines over time. From the interface, you can inspect logs and manage tasks, for example retrying a task in
```

Comment:
I think the previous syntax is correct? The UI provides two things:

1. In-depth views of two things
    a. Pipelines
    b. Tasks
2. Overview of pipelines over time

Maybe it would be clearer to split this into two sentences instead.